locallib.php

```diff
@@ -25,7 +25,7 @@
 
 defined('MOODLE_INTERNAL') or die;
 require_once($CFG->dirroot.'/local/ltiprovider/ims-blti/blti_util.php');
-require_once($CFG->dirroot.'/course/lib.php');
```

Comment:
Are you sure we don't need this anymore?

Reply:
Are you right, I restored it